external/include/protocols/webserver/http_server.h

```diff
@@ -316,20 +316,35 @@ int http_send_response(struct http_client_t *client, int status, const char *bod
  */
 int http_send_response_with_status_msg(struct http_client_t *client, int status, const char* status_message, const char* body, struct http_keyvalue_list_t *headers);
 
+/**
+ * @brief http_send_response_with_status() sends the response.
```

Comment:
we can mention in http_send_response_with_status_msg doxygen comments that this should not be used for binary data.
API names http_send_response_with_status_msg & http_send_response_with_status. user can't distinguish purpose of API without code.

Reply:
add the comment in header file